tests_e2e/tests/lib/retry.py

```diff
@@ -57,3 +57,23 @@ def retry_ssh_run(operation: Callable[[], Any]) -> Any:
                     raise
             log.warning("The operation failed, retrying in 30 secs.\n%s", e)
         time.sleep(30)
+
+
+def retry_if_not_found(operation: Callable[[], bool], attempts: int = 5) -> bool:
```

Comment:
"if_not_found" implies some kind of search, how about "retry_if_false". Let's also make the sleep time a parameter with a default value.